mmv1/products/iamworkforcepool/api.yaml

```diff
@@ -0,0 +1,124 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: IAMWorkforcePool
+display_name: Cloud IAM
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://iam.googleapis.com/v1/
+scopes:
+  - https://www.googleapis.com/auth/iam
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Identity and Access Management (IAM) API
+    url: https://console.cloud.google.com/apis/library/iam.googleapis.com/
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'targetLink'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: True
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+objects:
+  - !ruby/object:Api::Resource
+    name: 'WorkforcePool'
+    base_url: locations/{{location}}/workforcePools
+    self_link: locations/{{location}}/workforcePools/{{workforce_pool_id}}
+    create_url: locations/{{location}}/workforcePools?workforcePoolId={{workforce_pool_id}}
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: |
+      Represents a collection of external workforces. Provides namespaces for
+      federated users that can be referenced in IAM policies.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Manage pools':
+          'https://cloud.google.com/iam/docs/manage-workforce-identity-pools-providers#manage_pools'
+      api: 'https://cloud.google.com/iam/docs/reference/rest/v1/locations.workforcePools'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'location'
+        description: The location for the resource.
+        required: true
+        input: true
+        url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: 'workforcePoolId'
+        description: |
+          The name of the pool. The ID must be a globally unique string of 6 to 63 lowercase letters,
+          digits, or hyphens. It must start with a letter, and cannot have a trailing hyphen.
+          The prefix `gcp-` is reserved for use by Google, and may not be specified.
+        required: true
+        input: true
+        url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          Output only. The resource name of the pool.
+          Format: `locations/{location}/workforcePools/{workforcePoolId}`
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'parent'
+        description: |
+          Immutable. The resource name of the parent. Format: `organizations/{org-id}`.
```

Comment:
Should also be input: true as it is immutable

Reply:
fixed